webapp/components/youtube_video.jsx

```diff
@@ -120,6 +124,13 @@ export default class YoutubeVideo extends React.Component {
     }
 
     render() {
+        if (this.state.failed) {
+            return <div/>;
+        }
+        if (!this.state.loaded) {
+            return <div style={{height: '407px'}}/>;
```

Comment:
Where did you get this height from? It doesn't seem to match the height below. Can we also avoid the magic number here?